workers/default/xml/marc_xml_generator.py

```diff
@@ -3,15 +3,16 @@
 
 def generate_marc_xml(work_path, data, template):
     """
-    Merges the article data with the journal data for all articles. Then calls the generic generat_xml function
-    for all articles.
-    :param work_path: Path to put the generated XML
-    :param data: Journal metadata
-    :param template: Marc-XML template
+    Merges article data with the journal data, then calls the
+    generic generate_xml function for all articles.
+    :param str work_path: Path to put the generated XML
```

Comment:
Wollen wir uns mal darauf einigen, ob wir str oder string in docstrings benutzen?

Reply:
pro `str`

Reply:
Ist mir recht egal. Kannst du deine Vorliebe für str begründen?

Reply:
https://docs.python.org/2/library/functions.html#str

Reply:
Thx, Einverstanden